site/resources.md

```diff
@@ -20,33 +20,37 @@ Here you will find external resources about Velero, such as videos, podcasts, an
 
 * Kubecon NA 2019 - How to Backup and Restore Your Kubernetes Cluster - Annette Clewett & Dylan Murray, Red Hat:
 
-    <iframe title="How to Backup and Restore Your Kubernetes Cluster - Annette Clewett & Dylan Murray, Red Hat" width="560" height="315" src="https://www.youtube.com/embed/JyzgS-KKuoo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    {% youtube JyzgS-KKuoo %}
```

Comment:
this doesn't render for me https://github.com/jonasrosland/velero/blob/youtube-plugin/site/resources.md
Is that because the plugin is not being used

Reply:
Also is this only going to be rendered on the website and not in markdown?

Reply:
This is only for rendering on the website, not in Markdown. Markdown doesn't load any ruby plugins.

Reply:
I can see this working as expected here https://deploy-preview-2236--velero.netlify.com/resources/

Reply:
Thanks for clarifying!
I don't know if we want o mention that in the markdown file. But this LGTM to me!